### PR TITLE
get socket client id

### PR DESCRIPTION
**Pull request by Aychihuahuausa**

### CODE REVIEW

File: BE/src/socket/socketio.js [hunk 1]
```diff
@@ -54,15 +54,27 @@ const subscripeToAllEvent = () => {
                 }
             }
         });
+
+
+        notifier.on('requestCanceled', (data) => {
+            for (let [id, socket] of io.of("/").sockets) {
+                if (data. clientId == socket.username) {
```
Comment (heshambadawy) [inline BE/src/socket/socketio.js:61]: is there soce between object and it's properity   if (data. clientId

File: BE/src/socket/socketio.js [hunk 2]
```diff
@@ -54,15 +54,27 @@ const subscripeToAllEvent = () => {
...
+                    let currentSocket = socket;
```
Comment (heshambadawy) [inline BE/src/socket/socketio.js:62]: actualy u don't that assign , u have socket object

**Review by heshambadawy — COMMENTED** [review]
Validate my Comments

File: BE/src/socket/socketio.js (hunk 2 above)
Comment (Aychihuahuausa) [reply BE/src/socket/socketio.js:62]: I have only object of socket id so I have to get the sender socket object

Comment (Aychihuahuausa) [reply BE/src/socket/socketio.js:62]: its {"socketId":"userID"}

**Review by heshambadawy — APPROVED** [review]

**Pull request merged**